pkg/report/writer.go

```diff
@@ -41,7 +42,11 @@ func WriteResults(format string, output io.Writer, results Results, outputTempla
 	case "json":
 		writer = &JsonWriter{Output: output}
 	case "template":
-		tmpl, err := template.New("output template").Parse(outputTemplate)
+		tmpl, err := template.New("output template").Funcs(template.FuncMap{
+			"escapeString": func(input string) string {
+				return html.EscapeString(input)
```

Comment:
What if we use xml.EscapeText?
https://golang.org/pkg/encoding/xml/#EscapeText

In that case, `escapeXML` is preferred rather than `escapeString`.

Reply:
@knqyf263 done, updated tests passing